src/plugins/mission_raw/mission_raw_impl.cpp

```diff
@@ -146,6 +178,30 @@ void MissionRawImpl::download_mission_cancel()
     // TODO: Implement cancel.
 }
 
+void MissionRawImpl::upload_mission_async(const std::vector<std::shared_ptr<MissionRaw::MavlinkMissionItemInt> > &mission_raw, const MissionRaw::result_callback_t &callback)
+{
+    if (!_parent->does_support_mission_int()) {
+        LogWarn() << "Mission int messages not supported";
+        // report_mission_result(callback, Mission::Result::ERROR);
```

Comment:
This callback is required and should be implemented using `_parent->call_user_callback`.